element-messenger/README.md

```diff
@@ -1,6 +1,8 @@
 # Simple Element Messenger Integration
 
-Element is a client solution for the popular Matrix chat protocol. This extension allows to embed an Element client into your ownCloud environment.
+Element is a client solution for the popular Matrix chat protocol.
+This extension has two parts, the first one lets you embed a self-hosted Elements client into your ownCloud environment.
+The second is a widget for usage within Elements, that allows to directly share files with within chat rooms.
```

Comment:
```suggestion
The second is an Elements widget, which allows to directly share files inside chat rooms.
```

Reply:
> [...]One more thing - in the latest version of file picker, we're also including library build. That means it can be imported into Vue projects directly without using the web component. Maybe that would be something of an interest for you?

Yes, using the library build should be better. I think in a future release we can address this